src/cci/cci_ssl.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation.
+ * Copyright (c) 2020 CUBRID Corp
```

Comment:
Remove above Copyright, because it is newly created file. Just

` * Copyright (C) 2016 CUBRID Corporation`